build/variables/test.yml

```diff
@@ -5,4 +5,5 @@ variables:
   Arcus.ServiceBus.Topic.Worker.HealthPort: '42066'
   Arcus.Databricks.JobId: 9
   Arcus.AzureFunctions.ApplicationInsights.MetricName: 'Databricks Job Completed'
-  Arcus.AzureFunctions.Databricks.Http.Port: '5050'
\ No newline at end of file
+  Arcus.AzureFunctions.Databricks.HttpPort: '5050'
```

Comment:
Just curious, why are we using `HttpPort` instead of `Http.Port`?

Reply:
Well, I wanted to make a distinction between 'Http' the project HTTP trigger and 'Http' the protocol.
Because otherwise, we would have `Arcus.AzureFunctions.Databricks.Http.Port` and `Arcus.AzureFunctions.Http.Http.Port` which is probably not very useful.

Reply:
Ah ok. In that case I would just use `Arcus.AzureFunctions.Http.Port` but it's ok!